var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -9,6 +9,14 @@
 from spack.package import *
 
 versions = [
+    {
+        "version": "2024.1.2",
+        "cpp": {
+            "url": "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/b6f11fab-a0ff-4d44-a5a0-ed59d0fa971c/l_dpcpp-cpp-compiler_p_2024.1.2.504_offline.sh",
+            "sha256": "39f5468aaaac32bb2b35e6e3a27224594546ee53d835c289ad1fc7b4b760f91a",
+        },
+        "ftn": None,
```

Comment:
Everything works better when there is both a C and Fortran compiler. I think we should skip this compiler in the installer. Spack users can still use 2024.1.2 as an external where it is more obvious how to configure a mixed version toolchain.

Reply:
@rscohn2 Can you give a bit more context? Is 2024.1.2 shipping without a fortran compiler?

Reply:
2024.1.2 is a patch release addressing a small set of  issues and there is only a C compiler. For users who experience a problem with 2024.1, it will be simpler to stay on 2024.0 until 2024.2. is available. Or create a mixed version toolchain with ifx 2024.1.0 and icx 2024.1.2.

Reply:
@rscohn2 So what is the right way to do that? What do you suggest?

Reply:
We don't have to do anything. Even if we do not publish 2024.1.2, people can still use the compiler if they manually install it.